packages/cli/package.json

```diff
@@ -60,8 +61,8 @@
     "lint": "eslint src/**/*.ts",
     "test": "jest src/**/*.test.ts",
     "postpack": "shx rm -f oclif.manifest.json",
-    "posttest": "yarn lint",
-    "prepack": "yarn build && oclif manifest && oclif readme",
+    "posttest": "na run lint",
+    "prepack": "na run && oclif manifest && oclif readme",
```

Comment:
thinking: we could abstract this into an external JS that handles the package manager and prevents `na` from being exposed to `package.json` 💭